elementary/monitor/api/invocations/invocations.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+from elementary.clients.api.api import APIClient
+from elementary.monitor.api.invocations.schema import DbtInvocationSchema
+from elementary.utils.log import get_logger
+
+logger = get_logger(__name__)
+
+
+class InvocationsAPI(APIClient):
+    def get_last_invocation(self, type: str) -> DbtInvocationSchema:
```

Comment:
It felt right to me to split those methods into 3 different methods.
If you feel like this is over engineering (or code duplicating), I can merge those 3

Reply:
I like it 🙂